src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
+  grid-template-rows: auto;
+  min-width: 1900px;
```

Comment:
Umm. We calculated with Andrea that the minimum width for the whole main content area of the Entity Viewer should be 1600, although even that is intolerably high. 1900 is way too much!

Reply:
I'll change it to 1600px fixed width then

Reply:
:heavy_check_mark: